eth2near/contract_wrapper/src/eth_client_contract.rs

```diff
@@ -74,11 +74,11 @@ impl EthClientContract {
         self.contract_wrapper
             .call_change_method(
                 "init".to_string(),
-                init_input.try_to_vec().unwrap(),
+                init_input.try_to_vec().expect("Error on parse init_input"),
                 None,
                 None,
             )
-            .unwrap();
```

Comment:
This will still panic and won't return `Error`.

Reply:
Here it is Okey, because it is during initialization(one time at the start). So, in that case, we want the program to finish.

Reply:
Ok for me, then just the PR name is a little bit misleading.